aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -7,16 +7,37 @@
 namespace at {
 namespace native {
 
-Tensor quantize_linear_cpu(const Tensor& self, double scale, int64_t zero_point, ScalarType dtype) {
+Tensor quantize_linear_cpu(
+    const Tensor& self, double scale, int64_t zero_point, ScalarType dtype) {
   auto quantizer = make_per_tensor_affine_quantizer(scale, zero_point, dtype);
   return quantizer->quantize(self);
 }
 
+Tensor quantize_linear_per_channel_cpu(
+    const Tensor& self, const Tensor& scales, const Tensor& zero_points,
+    IntArrayRef axis, ScalarType dtype) {
+  TORCH_CHECK(scales.dim() == 1, "scale tensor must have dimension 1");
+  TORCH_CHECK(zero_points.dim() == 1,
+              "zero_points tensor must have dimension 1");
+  TORCH_CHECK(scales.numel() == zero_points.numel(),
+              "number of elements in scales and zero_points must match");
+  TORCH_CHECK(axis.size() == 1, "axis tensor must have size 1");
```

Comment:
nit - it's not a tensor. And change the message that it's not yet supported (is it right?)

Reply:
we don't know whether we'll support size != 1 but size 1 is only thing that we'll support now.